client/src/setupTests.ts

```diff
@@ -3,3 +3,24 @@
 // expect(element).toHaveTextContent(/react/i)
 // learn more: https://github.com/testing-library/jest-dom
 import "@testing-library/jest-dom/extend-expect";
+global.microapps = {
+  getIdentity: (request: any) =>
+    new Promise((resolve, reject) => {
+      const fakeIdentity = {
+        iss: "",
```

Comment:
What is the meaning of the parameters here?
iss/sub/aud/iat/exp?
Consider renaming or adding some comments?

Reply:
I am keeping our fields identical to Spot API's return json, will add comments